app/src/main/java/com/codesoom/assignment/exceptions/NotFoundEntityException.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.exceptions;
+
+public class NotFoundEntityException extends RuntimeException{
```

Comment:
이런 경우엔 보통 단어 순서를 이렇게 하는 것이 좀 더 읽기 좋겠죠.
```suggestion
public class EntityNotFoundException extends RuntimeException {
```